apps/omg_watcher/lib/web/serializers/error.ex

```diff
@@ -0,0 +1,27 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Watcher.Web.Serializer.Error do
+  @moduledoc """
+  Serializes data into JSON response format.
```

Comment:
Improve doc: "Serializes error's code and description provided in JSON response data field."